Makefile

```diff
@@ -20,11 +21,11 @@ tidy:
 build: $(MICROSERVICES)
 
 cmd/device-rfid-llrp-go:
-	$(GO) build $(GOFLAGS) -o $@ ./cmd
+	$(GOCGO) build $(GOFLAGS) -o $@ ./cmd
```

Comment:
This was not added by you, but we need to move the `build` target above the `tidy` target.

```makefile
build: $(MICROSERVICES)

tidy:
	go mod tidy
```

Reply:
fixed